openshift_cli_installer/cli.py

```diff
@@ -149,28 +128,30 @@
     multiple=True,
 )
 @click.option(
-    "--destroy-all-clusters",
+    "--destroy-clusters-from-s3-bucket",
     help="""
 \b
-Destroy all clusters under `--clusters-install-data-directory` and/or
-saved in S3 bucket (`--s3-bucket-path` `--s3-bucket-name`).
-S3 objects will be deleted upon successful deletion.
+Destroy clusters S3 bucket, --s3-bucket-name is required and optional --s3-bucket-path.
     """,
+    show_default=True,
     is_flag=True,
+)
+@click.option(
+    "--destroy-clusters-from-s3-bucket-query",
+    help="""
+\b
+Option to pass query to --destroy-clusters-from-s3-bucket, match only files that have it.
+    """,
     show_default=True,
 )
 @click.option(
-    "--destroy-clusters-from-s3-config-files",
-    help=f"""
+    "--destroy-clusters-from-install-data-directory",
+    help="""
 \b
-Destroy clusters from a list of paths to `{CLUSTER_DATA_YAML_FILENAME}` files.
-The yaml file must include `s3-object-name` with s3 objet name.
-`--s3-bucket-name` and optionally `--s3-bucket-path` must be provided.
-S3 objects will be deleted upon successful deletion.
-For example:
-    '/tmp/cluster1/,/tmp/cluster2/'
+Destroy clusters from cluster data files located at --clusters-install-data-directory
```

Comment:
example?

Reply:
What example? this is cli flag `--clusters-install-data-directory`

Reply:
Update the README